File: src/main/java/com/example/repository/CardRepository.java
package com.example.repository;

import com.example.entity.Card;
import org.springframework.data.jpa.repository.JpaRepository;
import org.springframework.stereotype.Repository;

import java.util.List;
import java.util.UUID;

@Repository
public interface CardRepository extends JpaRepository<Card, UUID> {
    public List<Card> findByOwner_Id(UUID uuid);
}

File: src/main/java/com/example/repository/CardRepository.kt
package com.example.repository

import com.example.entity.Card
import org.springframework.data.jpa.repository.JpaRepository
import org.springframework.data.redis.core.RedisHash
import org.springframework.stereotype.Repository
import java.util.UUID

@Repository
@RedisHash
interface CardRepository: JpaRepository<Card, UUID> {
    fun findByOwner_Id(id: UUID): List<Card>
}